react/features/polls/actions.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import {
+    END_POLL,
+    START_POLL,
+    VOTE_POLL,
+    POLL_SESSION_STARTED,
+    POLL_SESSION_VOTE,
+    POLL_SESSION_FINISHED
+} from './actionTypes';
+import { showNotification } from '../notifications/actions';
+
+/**
+ * Initiate a poll session.
+ *
+ * @param {Object} payload - Poll object.
+ * @returns {{
+ *      type: POLL_SESSION_INITIATED,
+ *      question: string,
+ *      items: Array<Object>
+ * }}
+ */
+export function startPoll(payload: Object) {
+    return {
+        type: START_POLL,
+        ...payload
```

Comment:
It's not clear to me what this payload needs to look like. I'd say either add a flow type or make sure to document the details of the payload, including what's needed in the nested objects.

Reply:
done